utils/src/deployer.rs

```diff
@@ -245,7 +245,7 @@ pub async fn deploy_light_client_contract<M: Middleware + 'static>(
 pub async fn deploy_mock_light_client_contract<M: Middleware + 'static>(
     l1: Arc<M>,
     contracts: &mut Contracts,
-    constructor_args: Option<(LightClientState, u32)>,
+    constructor_args: Option<(LightClientState, u32, u32)>,
```

Comment:
I think it makes the code hard to read to have tuples where a single type is used for multiple things.

Suggest creating a struct `LightClientConstructorArgs` with named fields and implement `Default` and `From<LightClientConstructorArgs> for (LightClientState, u32, u32)`.

Reply:
addressed here https://github.com/EspressoSystems/espresso-sequencer/commit/1efd5da955490c9670980a9e8380b2936c4cc759